QUEST_JOBSTEP.tsv

```diff
@@ -1638,260 +1638,260 @@ QUEST_JOBSTEP_20150317_001637	Krivis Master told you that Krivis siblings who we
-QUEST_JOBSTEP_20150317_001671	Swordsman Master thinks the courage to protect colleagues is most important. Provoke the monsters using the provoking orb at Penitence Route of Cathedral and defeat them.
-QUEST_JOBSTEP_20150317_001672	You have fulfilled the amount which Swordsman Master requested to you. Go back to Swordsman Master and report.
-QUEST_JOBSTEP_20150317_001673	Go find Highlander Master
-QUEST_JOBSTEP_20150317_001674	Go find Highlander Master in Klaipeda.
-QUEST_JOBSTEP_20150317_001675	Go find Peltasta Master
-QUEST_JOBSTEP_20150317_001676	Highlander Master told you that if you could help Peltasta Master, he will give you lessons. Go find Peltasta Master.
-QUEST_JOBSTEP_20150317_001677	Peltasta Master is in a hurry. Talk to Peltasta Master again.
+QUEST_JOBSTEP_20150317_001671	The Swordsman Master taught that the courage to protect allies is very important. Provoke the monsters using the provoking orb at Penitence Route of Cathedral and defeat them.
+QUEST_JOBSTEP_20150317_001672	You have fulfilled the amount which the Swordsman Master requested to you. Go back to the Swordsman Master and report.
+QUEST_JOBSTEP_20150317_001673	Go find the Highlander Master
+QUEST_JOBSTEP_20150317_001674	Go find the Highlander Master in Klaipeda.
+QUEST_JOBSTEP_20150317_001675	Go find the Peltasta Master
+QUEST_JOBSTEP_20150317_001676	The Highlander Master told you that if you could help the Peltasta Master, he will offer you some lesson. Go find the Peltasta Master.
+QUEST_JOBSTEP_20150317_001677	The Peltasta Master is in a hurry. Talk to the Peltasta Master again.
 QUEST_JOBSTEP_20150317_001678	Go meet Uska
-QUEST_JOBSTEP_20150317_001679	Peltasta Master told you that Uska received a request, but since he doesn't have time, he can't support it. Go meet Uska on behalf of Peltasta Master.
-QUEST_JOBSTEP_20150317_001680	You told him that you will help Uska on behalf of busy Highlander Master and Peltasta Master. Talk to Uska again.
-QUEST_JOBSTEP_20150317_001681	Defeat the source of the rumor at the 2nd Floor of Crystal Mines
-QUEST_JOBSTEP_20150317_001682	Uska told you that the work needs to start again at Crystal Mines, but he can't due to the gigantic monster at the 2nd Floor of Crystal Mines. Get rid of the source of the rumor at the 2nd Floor of Crystal Mines.
+QUEST_JOBSTEP_20150317_001679	The Peltasta Master told you that Uska received a request, but since he doesn't have time, he can't support it. Go meet Uska on behalf of the Peltasta Master.
+QUEST_JOBSTEP_20150317_001680	You told him that you will help Uska on behalf of the Highlander Master and Peltasta Master. Talk to Uska again.
+QUEST_JOBSTEP_20150317_001681	Defeat the source of the rumor at the 2nd floor of Crystal Mine
+QUEST_JOBSTEP_20150317_001682	Uska told you that the work needs to start again at Crystal Mine, but he can't due to the gigantic monster at the 2nd floor of Crystal Mine. Get rid of the source of the rumor at the 2nd floor of Crystal Mine.
```

Comment:
2nd floor could be abbreviated to 2F as I saw this ingame. 
e.g. Crystal Mine 2F